lib/options.json

```diff
@@ -11,16 +11,7 @@
       "type": "boolean"
     },
     "clientLogLevel": {
-      "enum": [
-        "info",
-        "warn",
-        "error",
-        "debug",
-        "trace",
-        "silent",
-        "none",
-        "warning"
-      ]
+      "enum": ["none", "error", "warn", "info", "log", "verbose"]
```

Comment:
Let's move it to `clientOptions`, i.e.:
```
clientOptions: { 
  logging: {
    level: 'info',
    // Let's avoid implement debug in this PR, just for future, schema should not contain debug too
    debug: [
      'MyPlugin',
      /MyPlugin/,
      (name) => name.contains('MyPlugin')
    ]
  }
}
```

Like here https://webpack.js.org/configuration/other-options/#infrastructurelogging

Reply:
@evilebottnawi Should we rename the `client-log-level` option then? Maybe `client-logging`? I think just `logging` might be confusing.

Reply:
@Loonride `webpack` has `infrastructureLogging`, so better to keep this same - https://webpack.js.org/configuration/other-options/

Reply:
I think we should move all client options to `clientOptions`, maybe we should rename `clientOptions` to `client`, less characters in the name, what do you think?

Reply:
Yes, I moved `logging` into `clientOptions`. I agree that it should be renamed to `client`, but that is an easy fix, I will do it later